network-api/networkapi/buyersguide/tests.py

```diff
@@ -393,6 +395,7 @@ def test_category_view(self):
         self.assertEqual(response.status_code, 200, 'The category "Smarth Home" should work by slug')
 
 
+@override_settings(DEBUG=True)
```

Comment:
`DEBUG=True` is too broad. Use `@override_settings(STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")` instead.

Reply:
yeah, this works for me too. Is there no way to use the Whitenoise backend and have it build the static file manifest on request?

Reply:
I'll leave this un-fixed for a little bit because if I update the code this comment thread will disappear. If there is no way to use whitenoise for local dev then I'll update it to use the contrib.staticfiles.storage backend.